brave/brave-core
# Moves Uphold contributions in db

Resolves https://github.com/brave/brave-browser/issues/8571

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- enable rewards
- connect KYC uphold account
- go to verified publisher
- tip publisher
- make sure that balance is reduced and that you see record in summary and tips table
- go to publisher_info_db and in `contribution_info` table you should see record that has 2 in `processor` field

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
